core/src/main/resources/config.yml

```diff
@@ -175,6 +175,12 @@ above-bedrock-nether-building: false
 # want to download the resource packs.
 force-resource-packs: true
 
+# A list of links to send to the client to download resource packs from.
+# These must be direct links to the resource pack, not a link to a page containing the resource pack.
+cdn-resource-packs:
+  # Example: GeyserOptionalPack
+  - "https://ci.opencollab.dev/job/GeyserMC/job/GeyserOptionalPack/job/master/lastSuccessfulBuild/artifact/GeyserOptionalPack.mcpack"
```

Comment:
poggers

Reply:
right? xd
Not sure though if it might make sense to comment it out, it is optional afterall

Reply:
afaik the only reason we weren't including it beforehand was because we didn't want to bundle mojang assets in the jar - this is no longer an issue